lib/notes_page.dart

```diff
@@ -1,72 +1,49 @@
 import 'package:flutter/material.dart';
+import 'package:classic/theme/theme_manager.dart';
+import 'package:classic/theme/theme_constants.dart';
 
-class NotesPage extends StatefulWidget {
-  @override
-  _NotesPageState createState() => _NotesPageState();
-}
-
-class _NotesPageState extends State<NotesPage> {
-  final TextEditingController _noteController = TextEditingController();
-  late FocusNode _focusNode;
-
-  @override
-  void initState() {
-    super.initState();
-    _focusNode = FocusNode();
-  }
+class NotesPage extends StatelessWidget {
+  final String name;
+  final ThemeManager themeManager;
 
-  @override
-  void dispose() {
-    _focusNode.dispose();
-    super.dispose();
-  }
+  const NotesPage({Key? key, required this.name, required this.themeManager}) : super(key: key);
 
   @override
   Widget build(BuildContext context) {
-    TextTheme _textTheme = Theme.of(context).textTheme;
     bool isDark = Theme.of(context).brightness == Brightness.dark;
 
     return Scaffold(
       appBar: AppBar(
-        title: Text("Notes Page"),
+        title: Text("Notes for $name"),
         backgroundColor: isDark ? Colors.grey[850] : Colors.white,
         foregroundColor: isDark ? Colors.white : Colors.black,
         actions: [
           IconButton(
-            icon: Icon(Icons.save),
+            icon: Icon(isDark ? Icons.wb_sunny : Icons.nights_stay),
```

Comment:
This button is not required all over the app, just have it in one place where you can change the theme